api/views.py

```diff
@@ -168,6 +176,28 @@ def create_hangout(*, group_size=HANGOUT_GROUP_LIMIT):
     return hangout
 
 
+@render_json(BrownbagSerializer)
+def create_brownbag(date):
+    """
+    Create a brownbag, maintaining a unique date for it.
+    """
+    users_queryset = User.objects.filter(
+        brownbag__isnull=True).values_list('id', flat=True)
+    rand = Randomizer(list(users_queryset))
+    try:
+        next_presenter_id = rand.get_random()
+    except IndexError:
+        # Cannot choose from an empty sequence since list is empty.
+        return "All users have been assigned a brownbag"
```

Comment:
We should probably `raise` here instead of returning a string. The `render_json` decorator will error out when it tries to create a `serializer` with `BrownbagSerializer` class given this string as this function's output.